### ISSUE TITLE
bm data

### ISSUE THREAD

**Issue by hnlyj** [JChen-UToronto/BEPS_D#1]
what are biological parameters  from readbm.c?

**Comment by chen-hm910** [comment]
您好，每日beps模型中输入的气象数据，应该是每天的累计值还是每天的平均值？

**Comment by chenjgeog** [comment]
should all be daily average values.
________________________________

**Comment by chen-hm910** [comment]
请问该模型可以模拟每日NEP变化吗？

**Comment by chenjgeog** [comment]
Yes, hourly BEPS can produce diurnal variations of NEP, and daily BEPS can calculate daily NEP. Which one you are using?

Jing M. Chen
Professor, CRC, FRSC
Department of Geography
University of Toronto
Sydney Smith Hall, Room 5058
100 St. George Street
Toronto, Ontario
Canada M5S 3G3
Tel: 416-978-7085
Fax: 416-946-3886
http://faculty.geog.utoronto.ca/Chen/Chen's%20homepage/home.htm
________________________________

**Comment by chen-hm910** [comment]
I'm using daily BEPS. Can the code in the web page calculate the NEP? I don't see it.
